jigsaw/client/credits_view.py

```diff
@@ -0,0 +1,58 @@
+from pygame import font, Rect, quit
+import pygame_gui
+from client.constants import screen, WHITE, BLUE, WIDTH, manager, HEIGHT
```

Comment:
```suggestion
from .constants import screen, WHITE, BLUE, WIDTH, manager, HEIGHT
```

Reply:
colors can be strings, drop those dumb color constants, plus the manager is an actual runtime object which shouldn't be passed around like that